src/css/profile/mobile/common/popup.less

```diff
@@ -5,19 +5,36 @@
 	display: none;
 	position: absolute;
 	left: 0;
-	width: 100%;
-	margin-top: 80 * @unit_base;
+	margin-bottom: 16 * @px_base;
 	z-index: (@z_base_popup + 1) !important;
 	background-color: @color_popup_bg;
-	border: 0;
+	border-radius: 26px;
 
-	.ui-popup-wrapper {
-		width: 100%;
+	@media (min-width: 0px) and (max-width: 479px) {
+		@media (min-height: 600px) and (max-height: 959px) {
+			width: 100%;
+			margin-top: 56 * @px_base;
+		}
 	}
 
-	@media (orientation: landscape) {
-		width: 720 * @unit_base;
-		left: 360 * @unit_base;
+	@media (min-width: 600px) and (max-width: 959px) {
+		@media (min-height: 0px) and (max-height: 479px) {
```

Comment:
I have a question.
Don't you check the orientation mode?

Reply:
applied.

done.